onnxruntime/core/providers/xnnpack/nn/max_pool.cc

```diff
@@ -5,26 +5,64 @@
 
 #include "core/graph/graph.h"
 #include "core/providers/utils.h"
-#include "core/providers/xnnpack/detail/utils.h"
+#include "core/framework/tensorprotoutils.h"
 
 // to sanity check output shape
 #include "core/framework/tensorprotoutils.h"
 
 namespace onnxruntime {
 namespace xnnpack {
-bool MaxPool::IsOnnxNodeSupported(const onnxruntime::Node& node, const GraphViewer& /*graph*/) {
+const onnx::TensorProto* GetQuantizationZeroPoint(const InitializedTensorSet& initializers,
+                                                  const NodeUnitIODef& io_def);
+const onnx::TensorProto* GetQuantizationScale(const InitializedTensorSet& initializers,
+                                              const NodeUnitIODef& io_def);
```

Comment:
What's the reason for these declarations? If something is defined elsewhere it should be made available via a header. They don't seem to be used in the latest implementation though.

Reply:
I want to make sure it has the same input/output quant scale and zeropoint at the begining, then I found QDQ selector has correspond checks. I will delete it.